packages/components/src/components/@deprecated/input/controller.ts

```diff
@@ -79,7 +79,8 @@ export class InputController extends ControlledInputController implements Watche
 		validateHideError(this.component, value, {
 			hooks: {
 				afterPatch: () => {
-					if (this.component.state._hideError) {
+					const hideError = this.component.state._hideError;
+					if (typeof hideError === 'boolean' && hideError) {
```

Comment:
Ich glaube hier brauchen wir eine Grundsatzentscheidung, wie wir mit der Regel `@stencil-community/strict-boolean-conditions` umgehen wollen.

Die jetzt umgesetzte Lösung finde ich ein wenig problematisch, weil sie die Logik verändert.
Die in meinen Augen "korrekte" Lösung um die Regel zufrieden zu stellen wäre im einfachsten Fall:
```
if (Boolean(this.component.state._hideError)) {
```

Das steht aber leider im Konflikt zu Prettier, dass den Boolean Constructor (m.M.n. zurecht) entfernt. Wir bräuchten z.B. eine weitere Konstante als Workaround.

(Alternativ könnten wir auch versuchen die Types zu fixen, so dass die State-Properties nicht mehr `unknown` sind, das wäre aber eine andere Baustelle.)